blueocean-core-js/src/js/sse/SseBus.js

```diff
@@ -44,7 +45,7 @@ export class SseBus {
         if (!this.sseListeners.job) {
             const sseListener = this.connection.subscribe('job', (event) => {
                 this._handleJobEvent(event);
-            });
+            }, { jenkins_org: AppConfig.getOrganizationName() });
```

Comment:
so how does this work - is this filtering events coming back or throwing away events not for the current org - if the latter, this wont' really be a scalable approach if there are lot of orgs right?